zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/ZipkinFactory.java

```diff
@@ -48,6 +49,9 @@
             .getLogger(ZipkinFactory.class);
     private static final String DEFAULT_ZIPKIN_SCRIBE = "127.0.0.1:9140";
 
+    private String bootstrapServers;
```

Comment:
I don't know how the configuration ends up looking.. would these be obviously scoped to kafka? (remembering there are multiple transports defined here)